bin/check-aggregate.rb

```diff
@@ -21,8 +21,14 @@ class CheckAggregate < Sensu::Plugin::Check::CLI
          short: '-a URL',
          long: '--api URL',
          description: 'Sensu API URL',
-         default: 'http://localhost:4567'
-
+         default:
+           if ENV['SENSU_API']
```

Comment:
would it make sense for this if line to be on the same line as the ```default:```

something like
```
    default: if ENV['SENSU_API']
                   blah
                 else
                    blah
                  end
```